.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -48,6 +56,15 @@ The launch tracker is meant to be an easy way of sharing a project's status with
 You should add a link to the launch tracker for this work here if applicable. One ideal template and instructions can be found here: bit.ly/amp-launch-tracker
 -->
 
+## Consideration checklist
+### Design
+- [ ] Design shows understanding of problem statement/priority
+- [ ] Acceptance/success criteria for problem being solved is clear
+- [ ] Design socialized
+<!--
+Mark as done once this design has been presented at a [Design Review](https://github.com/ampproject/amphtml/blob/532dbbbc15c879a32e26e1e890b3a6b5b71cb123/contributing/design-reviews.md).
+-->
+
 <!--
 Add anyone to this cc line that you want to notify about this I2I, including a reviewer once you have found one. See https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md for help in finding a reviewer.
```

Comment:
Will it be understood what's meant by "this cc line"? I don't see one written in the template. If not, could we incorporate a cc line into the visible part of the template?

Reply:
Oh, I see. The other template has

` /cc @ampproject/wg-approvers`

Should something similar be added here?

Reply:
Clarified the whole cc process:
- Added a link to help find a WG to cc
- Clarified that the Approvers WG is cced by default below.